inst/include/Rcpp/StringTransformer.h

```diff
@@ -1,7 +1,7 @@
 
 // clone.h: Rcpp R/C++ interface class library -- clone RObject's
 //
-// Copyright (C) 2010 - 2022 Dirk Eddelbuettel and Romain Francois
+// Copyright (C) 2010 - 2025 Dirk Eddelbuettel and Romain Francois
```

Comment:
Plus Inaki Ucar if you want.  Or leave as is with the year change.  Ditto for others.